### PR TITLE
Non zero exit on failure

### LINKED ISSUE #24: When the execution fails, the exit code is 0
I gave a try to this github action and the first execution was _green_.
I was amazed since I did not configure anything.

Looking at the logs, I could find the following last line:

```
fatal: could not read Username for 'https://github.com': No such device or address
```

Would it be possible to have a non 0 exit code when the job execution fails?

### PR DESCRIPTION
**Pull request by helaili**
This should fix #24

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -49,5 +49,4 @@ git add . && \
 git commit -m "jekyll build from Action ${GITHUB_SHA}" && \
 git push --force $remote_repo master:$remote_branch && \
 rm -fr .git && \
-cd ..
-exit 0
+cd .. && exit 0 || exit 1
```
Comment (mberlanda) [inline entrypoint.sh:52]: could it be an alternative just to `exit $?`
I am not sure that it would work out of the box

Comment (mberlanda) [reply entrypoint.sh:52]: do you have the possibility to add a test on this ? :smile:

Comment (mberlanda) [reply entrypoint.sh:52]: In order to test manually, you can replace the `exit` with `echo $?` and make sure that the error code is propagated across all commands.

If this is the case, you can just remove the final exit command.

I assumed that the `set -e` would already make the script fail, probably you are missing some options (e.g. `-o pipefail`)

Comment (helaili) [reply entrypoint.sh]: Thanks for the suggestion, it works nicely

**Pull request merged**